lib/osc.py

```diff
@@ -477,14 +473,14 @@ def createOrUpdateOStackVC(self, vc, update_vcid=None, useLegacyProvAttrs=False)
             data = self._isc_connection(method=method, url=url, body=body, action=action, headers=headers)
             if data is None:
                 self._output.log_error("Exit createOStackVC -- Failed to create new VC")
-                return(None)
+                ##return(None)
```

Comment:
remove the ##

Reply:
All changes done by rebase, edit, add, commit, fetch.